client/src/components/form-inputs/FormInputField.tsx

```diff
@@ -40,7 +40,7 @@ export const FormInputField = <T extends FieldValues>({
         value={value}
         variant={variant}
         error={invalid}
-        helperText={getFormErrorMessage(error?.type)}
+        helperText={error?.message || getFormErrorMessage(error?.type)}
```

Comment:
if custom message is passed, we use that first, otherwise default to common message

Reply:
We added this since for example on the login page, if you enter an invalid email it says:

```
Your input does not match the required pattern
```

which isn't useful to the user since they won't know what a pattern is

Instead it should be 

```
Please enter a valid email address
```

Reply:
I like this approach